go/tools/release/release.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"runtime"
+
+	"github.com/keybase/client/go/libkb"
+	gh "github.com/keybase/client/go/tools/release/github"
+)
+
+func githubToken() string {
+	token := os.Getenv("GITHUB_TOKEN")
+	if token == "" {
+		log.Fatal("No GITHUB_TOKEN set")
+	}
+	return token
+}
+
+func tag(version string) string {
+	return fmt.Sprintf("v%s", version)
+}
+
+var repo = flag.String("repo", "client", "Repository in keybase")
+var version = flag.String("version", libkb.VersionString(), "Version for tag")
+var src = flag.String("file", "", "Path to source file")
+var dest = flag.String("dest", "", "Path to destination file")
+
+func main() {
+	flag.Parse()
+
+	if len(flag.Args()) < 1 {
+		log.Fatal("Specify action")
+	}
+	action := flag.Arg(0)
+
+	switch action {
+	case "version":
+		fmt.Printf("%s", libkb.VersionString())
+	case "increment-build":
+		var file = flag.String("file", "", "Path to version file")
+		err := libkb.WriteVersion(libkb.Version, libkb.Build+1, *file)
```

Comment:
The version "action" above it to output the version as evaluated by go (instead of grepping source).